packages/noco-docs/docs/020.getting-started/050.self-hosted/020.environment-variables.md

```diff
@@ -59,8 +59,9 @@ For production use-cases, it is **recommended** to configure
 | NC_S3_REGION                       | For S3 storage plugin - AWS S3 region                                                                                                                                                                                                       |                                                                                                |
 | NC_S3_ACCESS_KEY                   | For S3 storage plugin - AWS access key credential for accessing resource                                                                                                                                                                    |                                                                                                |
 | NC_S3_ACCESS_SECRET                | For S3 storage plugin - AWS access secret credential for accessing resource                                                                                                                                                                 |                                                                                                |
-| NC_ADMIN_EMAIL                     | For updating/creating super admin with provided email and password                                                                                                                                                                          |                                                                                                |
 | NC_ATTACHMENT_FIELD_SIZE           | For setting the attachment field size(in Bytes)                                                                                                                                                                                             | Defaults to 20MB                                                                               |
+| NC_MAX_ATTACHMENTS_ALLOWED         | Maximum Number of attachments per cell                                                                                                                                                                                                      |                                                                                                |
```

Comment:
FYI. `NC_MAX_ATTACHMENTS_ALLOWED` is not used for OSS but EE version.

Reply:
Ah, so it has no effect on the official `nocodb/nocodb` Docker image?

Does a maximum number of attachments also apply if an external object storage provider (e.g. Backblaze B2) is configured?

Reply:
oh wait. seems i might mix up something. lemme check with the team first and get back to you later.

Reply:
Confirmed with the team. This should be included in OSS as well.

Reply:
Glad to hear! :+1:

Reply:
@wingkwong Does the limit apply regardless whether internal storage or external object storage (S3) is used?

Reply:
Correct. It's just the limit of files to be uploaded in one cell.